<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(72,35,60);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>100</y>
      <width>600</width>
      <height>450</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>450</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>450</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>40</y>
       <width>205</width>
       <height>45</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>205</width>
       <height>45</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>205</width>
       <height>45</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;Times&quot;;
color: rgb(10, 54, 157);
</string>
     </property>
     <property name="text">
      <string>Create Account</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>240</width>
       <height>240</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>240</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>240</width>
       <height>240</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>120</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;Times&quot;;
color: rgb(10, 54, 157);
</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_Username">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>140</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>220</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(150, 172, 183);
font: 13pt &quot;Times&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Times'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.SF NS Text';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>190</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>110</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;Times&quot;;
color: rgb(10, 54, 157);
</string>
     </property>
     <property name="text">
      <string>Contraseña</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_Password">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>210</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>220</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(150, 172, 183);
color: rgb(255, 255, 255);
font: 13pt &quot;Times&quot;;</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Times'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.SF NS Text';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_CreateAccount">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>350</y>
       <width>165</width>
       <height>32</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>165</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>165</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(10, 54, 157);
font: 14pt &quot;Times&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Crear cuenta nueva</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>260</y>
       <width>115</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>115</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>115</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;Times&quot;;
color: rgb(10, 54, 157);
</string>
     </property>
     <property name="text">
      <string>Suscripción</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Suscripciones">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>280</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(150, 172, 183);
color: rgb(255, 255, 255);
font: 13pt &quot;Times&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>Escoge una opción</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mensual</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Semestral</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Anual</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
